paperqa/agents/env.py

```diff
@@ -94,6 +95,11 @@ def settings_to_tools(
     return tools
 
 
+ALLOW_PARALLEL_TOOL_CALLS: bool = os.environ.get(
+    "PQA_ALLOW_PARALLEL_TOOL_CALLS", ""
+).lower() in {"true", "1"}
+
+
```

Comment:
How about we just YAGNI this and remove it? If we want to explore later, we can make it a setting.

Reply:
I don't feel strongly abt it -- can delete it